### PR TITLE
Add blogpost for 0.16.0-RC3 and 0.15.0 - (Scala Days Release)

### PR DESCRIPTION
**Pull request by biboudis**
Merge on *Tuesday 11/06/2019, 9:00am*

### CODE REVIEW

File: docs/blog/_posts/2019-06-11-16th-dotty-milestone-release.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+<!-- https://github.com/lampepfl/dotty/pulls?q=is%3Apr+closed%3A%3E2019-05-23+is%3Aclosed+sort%3Acomments-desc -->
+
+## Syntax Change: Type Lambdas
+
+We reconsider the syntax of type lambdas in an effort to provide an improved
+visual cue for two categories of types: types that relate to normal function
+types and types that operate on a higher level. The _fat_ arrow `=>` definitely
+relates to the first, while we reserve now `->` to mean _pure function_ in the
+future. As a result, we disengage `=>` from type lambdas, whice are now
```
Comment (Jentsch) [inline docs/blog/_posts/2019-06-11-16th-dotty-milestone-release.md]:
```suggestion
future. As a result, we disengage `=>` from type lambdas, which are now
```

**Comment by anatoliykmetyuk** [comment]
Due #6642 we will be 0.16.0-RC2 and not 0.16.0-RC1 this time – please consider this in the article

File: docs/blog/_posts/2019-06-11-16th-dotty-milestone-release.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+## Polymorphic function types
+
+We add preliminary support for _polymorphic function types_. Nowadays, when we
+want to write a universally quantified function over elements of lists of type
+`T` we write e.g., `List[T] => List[(T, T)]` where `T` is bound at an enclosing
+definition. With polymorphic function types (PFT hereafter) we can quantify the
+parametric type locally. For example:
+
+```scala
+[T <: AnyVal] => List[T] => List[(T, T)]
```
Comment (azolotko) [inline docs/blog/_posts/2019-06-11-16th-dotty-milestone-release.md]: Should this be `[T <: AnyVal] =>> List[T] => List[(T, T)]`? I mean, the expression starts with a type lambda, doesn't it?

**Comment by anatoliykmetyuk** [comment]
@biboudis the contributors are available in the release notes: https://github.com/lampepfl/dotty/releases/tag/0.16.0-RC3

File: docs/blog/_posts/2019-06-11-16th-dotty-milestone-release.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+author: Aggelos Biboudis
+authorImg: /images/aggelos.png
+date: 2019-06-11
+---
+
+Hello again! Today, we are excited to announce the 16th release of Dotty. The
+development of Dotty continues according to our schedule but today, Tuesday June
+the 11th, we are electrified as it is the first day of [Scala Days 2019](https://scaladays.org/)
+which marks the *10th* anniversary of Scala Days.
+With this release we are getting closer to the _envelop_ of the new features
```
Comment (bishabosha) [inline docs/blog/_posts/2019-06-11-16th-dotty-milestone-release.md]: _envelope_

**Comment by biboudis** [comment]
Added your name. I don't know how to put both images so I removed mine.

**Pull request merged**